### PR TITLE
Fix account console tests for dev

### LINKED ISSUE #27604: Account console dev environment broken
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

account/ui

### Describe the bug

Account console tests cannot be run due to changes in #22192:
- Keycloak server is supposed to run on port [8080](https://github.com/keycloak/keycloak/blob/2199d37879dbbcda085175f59233083c5f609c13/js/apps/account-ui/test/admin-client.ts#L12), but dev server runs on [8180](https://github.com/keycloak/keycloak/blob/2199d37879dbbcda085175f59233083c5f609c13/js/apps/keycloak-server/scripts/start-server.js#L42)
- Playwright tests fail since the [router is registered to `/realms/master/account`](https://github.com/keycloak/keycloak/blob/2199d37879dbbcda085175f59233083c5f609c13/js/apps/account-ui/src/main.tsx#L19-L20) but the tests use also other realms than `master`

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Tests are runnable in dev environment

### Actual behavior

Tests fail

### How to Reproduce?

Run playwright tests (`pnpm run tests`) in account-ui

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by hmlnarik**
Fixes: #27604

### CODE REVIEW

File: js/apps/account-ui/test/utils.ts [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import { getRootPath } from "../src/utils/getRootPath";
 
-export function getBaseUrl(): string {
+export function getTestServerUrl(): string {
   return process.env.KEYCLOAK_SERVER ?? "http://localhost:8080";
 }
 
+export function getKeycloakServerUrl(): string {
```
Comment (jonkoops) [inline js/apps/account-ui/test/utils.ts:7]: Let's keep this under the name `getBaseUrl()`. I think that is explanatory enough.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
@hmlnarik exactly which tests are failing locally without these fixes? The Account Console doesn't support dynamically changing realms, so I would expect any realm change to be a full page load, setting the `basename` at that time.

File: js/apps/account-ui/test/utils.ts (hunk 1 above)
Comment (hmlnarik) [reply js/apps/account-ui/test/utils.ts:7]: "Base" can refer either to account console server or keycloak server, or the base URL of the two if console and REST endpoints are served by the same server. I'd rather keep this name unambiguous.

File: js/apps/account-ui/src/routes.tsx [hunk 2]
```diff
@@ -57,10 +57,8 @@ export const PersonalInfoRoute: IndexRouteObject = {
   element: <PersonalInfo />,
 };
 
-export const RootRoute: RouteObject = {
-  path: "/",
-  element: <Root />,
-  errorElement: <ErrorPage />,
+export const InRealmRoute: RouteObject = {
```
[the remark this answers is not in the document]
Comment (hmlnarik) [reply js/apps/account-ui/src/routes.tsx:60]: I was unable to find the way to do so but now I seem to have found the right syntax.

File: js/apps/account-ui/test/utils.ts (hunk 1 above)
Comment (jonkoops) [reply js/apps/account-ui/test/utils.ts:7]: How about `getOrigin()`? That is a [well defined term](https://developer.mozilla.org/en-US/docs/Web/API/URL/origin) .

Comment (hmlnarik) [reply js/apps/account-ui/test/utils.ts:7]: Not sure.. Origin does not specify if it is the REST endpoint origin or account console origin.

File: js/apps/account-ui/src/routes.tsx [hunk 3]
```diff
@@ -72,6 +70,8 @@ export const RootRoute: RouteObject = {
     ResourcesRoute,
     ContentRoute,
   ],
+  element: <Root />,
+  errorElement: <ErrorPage />,
```
Comment (jonkoops) [inline js/apps/account-ui/src/routes.tsx:74]: Just a small nit, but let's leave these properties on top where they used to be.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**